chapter12/qli/ex3.pl

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+
+foreach my $file (@ARGV) {
+  if( -e $file ){
+    print "$file is readable writable and owned by me\n" if ( -r -w -O $file );
```

Comment:
You could have used the underscore symbol here, rather than creating a new handle to `$file`